### PR TITLE
add initial content for release consolidation

### PR DESCRIPTION
**Pull request by Siegfriedk**
This PR combines the content of the website and tractus-x-release repository into sig-release.

It thereby fixes the [Issue #572](https://github.com/eclipse-tractusx/sig-release/issues/572)

See:
- https://github.com/eclipse-tractusx/tractus-x-release/pull/26
- https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/852

Superseeds https://github.com/eclipse-tractusx/sig-release/pull/519

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,94 +1,27 @@
...
-is used to align the overarching Tractus-X releases.
+To make adopting the latest [Catena-X Standards](https://catena-x.net/de/standard-library) easier, reference implementations are provided through the Eclipse Tractus-X Project
 
-### How can I get involved
+## Release Cycle - major vs. minor release
 
-In case you experienced a bug, unexpected behaviour, or you want to propose enhancements to Eclipse Tractus-X,
-feel free to use one of the provided [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose) and describe your request.
-Please be aware, that not every feature request can be integrated and that we also cannot treat every issue with the highest priority.
+Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release tractus-x-release.md following [CalVer](https://calver.org) is released.
```
Comment (RolaH1t) [inline README.md:13]: Tractus-X operates on a quarterly release cycle. Each quarter, a new [tractus-x release] (tractus-x-release.md) following [CalVer](https://calver.org) is released.

File: docs/product_release.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+Product releases are done by creating a GitHub release. Every release is following the [semantic versioning](https://semver.org/) pattern and must be marked in the git history via tag.
```
Comment (RolaH1t) [inline docs/product_release.md:1]: in general: pay attention to CAPITAL or small letters => especially when referencing URLs and documents

File: docs/product_release.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Patching strategy
+tbd
+
+Helm Repository
+For information about using the Tractus-X Helm repository, please refer to the charts repository.
+
+
+## Versioning
+
+- Semantic Versioning
```
Comment (RolaH1t) [inline docs/product_release.md:52]: consider to add semver URL

File: docs/tractus-x-release.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
+## Intro
+
+Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release following CalVer is released. Each Tractus-X release contains multiple products product_release.md.
```
Comment (RolaH1t) [inline docs/tractus-x-release.md:3]: Tractus-X operates on a quarterly release cycle. Each quarter, a new tractus-x release following [CalVer] (URL) is released. Each tractus-x release contains multiple [products] (product_release.md).

File: docs/tractus-x-release.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+- Planning
+- Feature Work and Product Release
+- Quality assurance in parallel
+- Opt-In for participation in TX Release
+- Completion of Q-Assurance
+- TX Release
+
+Once feature work on product level is initiated, the quality assurance of the foreseen release package runs in parallel. The vehicle of Umbrella helm charts is used to continuously test pre-defined product bundles. Automatically and environment-less.
+
+Reference onion rings: (add URL)
```
Comment (RolaH1t) [inline docs/tractus-x-release.md:38]: this helpful graph shall soon be published within a document of the CX Association => we can add the URL as soon as available

File: docs/tractus-x-release.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+Example for cxOS (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf):
```
Comment (RolaH1t) [inline docs/tractus-x-release.md:40]: Example for the Catena-X Operation System ("cxOS")

File: docs/tractus-x-release.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+A configured umbrella helm chart starts-up all relevant products. No matter if carry-over versions of a previous TX release or evolutions of the current iteration. User Journey tests (e2e) are automatically performed - customized for the respective bundle of products and within given boundary conditions.
```
Comment (RolaH1t) [inline docs/tractus-x-release.md:42]: No matter if carry-over product versions of a previous TX release or product evolutions, developed within the current iteration.

File: docs/tractus-x-release.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+Any findings are looped back to the product development teams. Product teams are obliged to fix any defects which prevent successful testing of the bundle.
```
Comment (RolaH1t) [inline docs/tractus-x-release.md:44]: Any findings are looped back to the [product development teams] (product_release.md)

File: docs/tractus-x-release.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+- Overarching architecture documentation (if available)
+
+Tractus-X offers a central Helm chart repository. It supports two channels for released Helm charts - dev and stable.
+The dev-channel is used to publish the most recently released charts. It is updated nightly and automatically pulls in the latest chart releases of the eclipse-tractusx GitHub organization.
+The stable channel is used by the release management group, to publish all helm charts, that were successfully tested and included in an overarching release. This means, that the stable channel only includes specific versions of product charts, that are tested to the best of our knowledge to work together with other stable charts.
+
+
+## Versioning
+
+- Calendar versioning
```
Comment (RolaH1t) [inline docs/tractus-x-release.md:110]: add URL

File: docs/planning.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
+# Roadmap and Release Process
```
[the remark this answers is not in the document]
Comment (Siegfriedk) [reply docs/planning.md:1]: Not sure if i get it? I assumed its english rule to have capital letters in a title. If you have specific feedback, pls provide it as i'm not a english grammer expert at all and need to lookup those rules adhoc

File: README.md (hunk 1 above)
Comment (Siegfriedk) [reply README.md:13]: I don't think thats correct. Tractus-X is a name and we are referenzing a document with the same name. I fixed the link

File: docs/product_release.md (hunk 2 above)
Comment (Siegfriedk) [reply docs/product_release.md:1]: You mean Semantic Versioning? or something else?

File: docs/product_release.md (hunk 3 above)
Comment (Siegfriedk) [reply docs/product_release.md:52]: added url

File: docs/tractus-x-release.md (hunk 4 above)
Comment (Siegfriedk) [reply docs/tractus-x-release.md:3]: As mentioned in the other comment, i would write Tractus-X upper case.

Also pls be more specific what issues you see. Its weird to read and diff the sentence to find this out

Comment (Siegfriedk) [reply docs/tractus-x-release.md:3]: Btw. we are repeating ourselves here. Its the same sentence as in README.md

File: docs/tractus-x-release.md (hunk 6 above)
Comment (Siegfriedk) [reply docs/tractus-x-release.md:40]: I added the text, fixed the link template and shortend it to 231006_Whitepaper_Tractus-X.pdf

File: docs/tractus-x-release.md (hunk 7 above)
Comment (Siegfriedk) [reply docs/tractus-x-release.md:42]: So we reference Tractus-X now as TX? Shouldn't we just keep Tracuts-X? Its short enough

Comment (Siegfriedk) [reply docs/tractus-x-release.md:42]: Text adjusted to suggestion

File: docs/tractus-x-release.md (hunk 8 above)
Comment (Siegfriedk) [reply docs/tractus-x-release.md:44]: product_release.md doesn't contain any real description of a product development team. Why would we link to this?

File: docs/tractus-x-release.md (hunk 9 above)
Comment (Siegfriedk) [reply docs/tractus-x-release.md:110]: done

File: README.md (hunk 1 above)
Comment (RolaH1t) [reply README.md]: pls update to:
Tractus-X operates on a quarterly release cycle. Each quarter, a new [Tractus-X release] (tractus-x-release.md) following [CalVer](https://calver.org/) is released.
=> mind the necessary URL link in normal brackets

File: docs/planning.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (RolaH1t) [reply docs/planning.md:1]: thanks for checking / resolved

File: docs/product_release.md (hunk 2 above)
Comment (RolaH1t) [reply docs/product_release.md]: thanks for checking / resolved

File: docs/tractus-x-release.md (hunk 4 above)
Comment (RolaH1t) [reply docs/tractus-x-release.md]: no duplicates found. pls keep sentence.
pls update to:
Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release following [CalVer] (URL) is released. Each Tractus-X release contains multiple [products] (product_release.md).
hint: URL links shown in normal brackets

File: docs/tractus-x-release.md (hunk 5 above)
Comment (RolaH1t) [reply docs/tractus-x-release.md:38]: still trying to obtain URL

File: docs/tractus-x-release.md (hunk 7 above)
Comment (RolaH1t) [reply docs/tractus-x-release.md]: compromise proposal:
No matter if carry-over versions of a previous Tractus-X release or evolutions of the current iteration. User Journey tests (E2E) are automatically performed - customized for the respective bundle of products and within given boundary conditions.

File: docs/tractus-x-release.md (hunk 8 above)
Comment (RolaH1t) [reply docs/tractus-x-release.md:44]: I still vote for my original proposal with relevant argumentation.
Leave the final call with you.

File: README.md (hunk 1 above)
Comment (Siegfriedk) [reply README.md]: done

File: docs/tractus-x-release.md (hunk 5 above)
Comment (Siegfriedk) [reply docs/tractus-x-release.md:38]: @RolaH1t url still missing; i keep it removed. We can do a patch if you get the url

File: docs/tractus-x-release.md (hunk 8 above)
Comment (Siegfriedk) [reply docs/tractus-x-release.md:44]: added the link

**Comment by stephanbcbauer** [comment]
Like this PR and the description. Since the lifecycle management changed a little (Maybe not for Tractus-X, but I would like to check) and also the standards are not provided on the “expected” URL any more, I would like to review this PR ... but ... not this week. It would be planned for next week.

**Comment by stephanbcbauer** [comment]
@Grand-Thibault maybe we can review this together because some parts have to match the working model (at least the open-source part :))

File: README.md [hunk 11]
```diff
@@ -1,94 +1,27 @@
...
-Every Release planning will be kicked off by two public alignment sessions. The dates and further details will be shared via
-[tractusx-dev](https://accounts.eclipse.org/mailing-list/tractusx-dev) mailing list.
-In addition to that, you can also find public meetings and info about how to join on our
-[Open Meetings](https://eclipse-tractusx.github.io/community/open-meetings) community page. 
-Issues or bug reports, that should be discussed in these meetings, have to be opened prior to the meeting via
-our [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose).
+Each Tractus-X release contains [multiple products](docs/product_release.md).
 
-### What can I expect
+Once per year, Tractus-X releases a major release, making the remaining three releases minor releases. A major release may contain critical breaking changes that have a major impact on data space participants, such as changes to enablement services. A minor release contains backward compatible functionality. In addition, patch versions provide backward compatible bug and security fixes.
```
Comment (lgblaumeiser) [inline README.md:17]: As I am working in the release management group in the association, the impact of the breaking change release thing is currently detailed. The clear requirement is, that the dataspace runs smoothly and companies can update in a defined grace period. This is defining "breaking changes" on a conceptual level. It has to be detailed, what this means for concrete developments. It is not possible to have breaking changes on the dataspace level, i.e., in the interactions companies have with other companies and especially with the operating company. So this statement here will break this requirement. Please wait until clarification is agreed upon.

**Review by lgblaumeiser — CHANGES_REQUESTED** [review]

File: README.md [hunk 12]
```diff
@@ -1,94 +1,27 @@
...
 
-We really welcome every contribution. Every Bug report and feature proposal takes time to prepare,
-is valuable to our project, and we very much appreciate this input.
-We are giving our best to give a first feedback in one week.
-If we should miss that, please stick with us and just use the commenting function to remind us of the issue.
+Please check the Change Log(s) for content, known knowns, and backward compatibility since the [calendar versioning scheme](https://calver.org) shows the year and month of release.
```
Comment (tomaszbarwicki) [inline README.md:19]: Perhaps we can link to general tractus-x release changlog: https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md ?

File: docs/product_release.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
+Product releases are done by creating a GitHub release. Every release is following the [Semantic Versioning](https://semver.org/) pattern and must be marked in the git history via tag.
```
Comment (tomaszbarwicki) [inline docs/product_release.md:1]: I'd suggest to add a title (top level heading) to align to other docs
```
# Product Release
## Intro
```

File: docs/tractus-x-release.md [hunk 14]
```diff
@@ -0,0 +1,129 @@
+## Intro
```
Comment (tomaszbarwicki) [inline docs/tractus-x-release.md:1]: Also adding doc title:

```
# Tractus-X Release
```

**Review by tomaszbarwicki — DISMISSED** [review]
Looks good to me in general, just few minor suggestions.

File: README.md (hunk 11 above)
Comment (stephanbcbauer) [reply README.md:17]: fine for me... @lgblaumeiser lets sync after the PR is merged. Would be happy to contribute here

File: docs/product_release.md (hunk 13 above)
Comment (Sicaine) [reply docs/product_release.md:5]: done

File: docs/tractus-x-release.md (hunk 14 above)
Comment (Sicaine) [reply docs/tractus-x-release.md:3]: done

File: docs/tractus-x-release.md (hunk 7 above)
Comment (Sicaine) [reply docs/tractus-x-release.md]: done

File: README.md (hunk 11 above)
Comment (lgblaumeiser) [reply README.md:17]: Is ok, as the impact of the sentence is still quite far in the future.

Comment (Siegfriedk) [reply README.md:17]: @lgblaumeiser i changed sooo much, pls let us merge it now and finetune it next week with @stephanbcbauer

**Review by tomaszbarwicki — APPROVED** [review]
